lib/roku/index.js

```diff
@@ -0,0 +1,53 @@
+var assert = require('assert');
+var https = require('../https');
+
+var apiUrls = {
+	production: 'https://apipub.roku.com/listen/transaction-service.svc/validate-transaction'
+};
+
+exports.verifyPayment = function (payment, cb) {
+	try {
+		assert.equal(typeof payment.devToken, 'string', 'Developer ID must be a string');
+		assert.equal(typeof payment.receipt, 'string', 'Receipt must be a string');
+		assert.equal(payment.receipt.match(/[\d|\w]/g).length, 32, 'Receipt must contain 32 digits');
```

Comment:
Ahh that's right, \w is alphanumeric --my bad. I'll adjust it.

Regarding length, I wanted to check if there was 32 alphanumeric characters in the string. By doing the quantifier, I think it would only match a range or up-to a certain point. I think doing the length on the match gives us the ability to check for 32 alphanumeric characters while ignoring the 4 dashes that are required in the next assertion.